tools/ci-build/publisher/src/subcommand/yank_release.rs

```diff
@@ -47,8 +62,18 @@ pub async fn subcommand_yank_release(
     }
     .context("failed to retrieve information about the release to yank")?;
 
+    let tag = release
+        .tag
+        .as_ref()
+        .ok_or_else(|| {
+            anyhow!("Versions manifest doesn't have a release tag. Can only yank tagged releases.")
```

Comment:
In what cases could this occur? If we did a manual release and never tagged it?

Reply:
It can happen if you download a generated SDK from CI or something and try to yank with its version.toml.